OWNERS

```diff
@@ -6,3 +6,9 @@ approvers:
 - autoscaling-wg-leads
 - networking-wg-leads
 - knative-release-leads
+
+dir_denylist:
+# Ignore OWNERS files in vendor (so we can stop stripping them out)
+- ^vendor/
```

Comment:
Is this needed? The description of `owner_dir_denylist` implies it might not be: https://github.com/kubernetes/test-infra/blob/eeb8ee3f4c97ab361daf29b521596498114d0593/prow/config/prow-config-documented.yaml#L535